models/marts/core/int_all_graph_resources.sql

```diff
@@ -53,6 +53,7 @@ joined as (
         unioned_with_calc.resource_type, 
         unioned_with_calc.file_path, 
         unioned_with_calc.directory_path,
+        unioned_with_calc.directory_path like 'models/%' as is_in_models_directory,
```

Comment:
Is there a way to get the `model-paths` from dbt_project.yml file instead of hard-coding?

Reply:
Nice! That's better than hard-coding, so I'll take it :)